### ISSUE TITLE
QA Ph4: Accessibility_Live modal is essentially inaccessible using keyboard only navigation 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#905]
Using only the keyboard to navigate the page, the live modal is essentially inaccessible.  User can expand the modal and close it, but cannot view any contents besides what is displayed upon the initial launch, nor can alternate tabs be selected. 

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Using the keyboard only to navigate the page, tap the tab key until landed on the live modal icon 
3. Expand the modal by tapping the return key 
4. Continue to tap the tab key and try the arrow keys to access modal content 
5. Note that modal is inaccessible 

Chrome 50 Mac OS X 10.11 

![gio_905](https://cloud.githubusercontent.com/assets/7585604/15231153/4ca4575a-184e-11e6-8993-9885f00186f0.png)
